backend/coins/eth/etherscan/etherscan.go

```diff
@@ -323,7 +330,7 @@ func (etherScan *EtherScan) Transactions(
 	result := struct {
 		Result []*Transaction
 	}{}
-	if err := etherScan.call(params, &result); err != nil {
+	if err := etherScan.call(context.TODO(), params, &result); err != nil {
```

Comment:
the linter was forcing me to pass the context down the stack (which I think makes absolutely sense!) but there are two places (this function and another one) in which we don't have it, so I'm open to suggestions here